src/GitHubCreateMergePRs/config.xml

```diff
@@ -11,17 +11,14 @@
 
     <!-- Roslyn non-vs-deps branches (flowing between releases) -->
     <merge from="release/dev16.10" to="main" />
-    <merge from="main" to="release/dev17.0" />
```

Comment:
I removed the flow to the dev17 branch.  Once we snap for a p1 we can add flow back (from d16 -> d17 -> main)